tests/conftest.py

```diff
@@ -14,13 +15,45 @@ def remove_headers(response: dict):
         response["headers"] = {}
         return response
 
+    def scrub_requests(request: dict):
+        if request.body:
+            try:
+                body_str = request.body.decode("utf-8")
+                body_json = json.loads(body_str)
+                if "token" in body_json:
+                    body_json["token"] = "ntn_..."
+                if "code" in body_json:
+                    body_json["code"] = "..."
+                if "redirect_uri" in body_json:
+                    body_json["redirect_uri"] = "http://..."
+                request.body = json.dumps(body_json).encode("utf-8")
+
+            except (json.JSONDecodeError, AttributeError):
+                pass
```

Comment:
Why do we need to silence the exception here? Is it expected that we have a JSON decode error or attribute error and do not care at all?